components/popover_list_members/popover_list_members.jsx

```diff
@@ -100,7 +100,8 @@ export default class PopoverListMembers extends React.PureComponent {
     render() {
         const isDirectChannel = this.props.channel.type === Constants.DM_CHANNEL;
 
-        const items = this.props.sortedUsers.map((user) => (
+        const activeUsers = this.props.sortedUsers.filter((user) => user.delete_at === 0);
```

Comment:
minor nitpick (non blocking): since 0 is a value used to detect a especific state and it is outside of the actual expected values, I'd change that into a named constant that expreses such meaning (something like "NOT_DELETED" or "ACTIVE")

Reply:
IMO, this is fine not to be refactored. Actually "NOT_DELETED = 0" makes it even more confusing IMO.

Nevertheless, I will make the change. But I note that `delete_at === 0` is being used all over the code. We should change all of them to be consistent.

Reply:
IMHO, I think a predicate function could help with clarity:

```ts
const NOT_DELETED_TIMESTAMP = 0;

export function isActive({delete_at}: User): boolean {
    return delete_at === NOT_DELETED_TIMESTAMP;
}

//...

import {isActive} from 'somewhere/utils/user';

const activeUsers = this.props.sortedUsers.filter(isActive);

```

That way any confusion behind the `0` constant contained in a clear `isActive` function.

Reply:
Done. IMO, if we are refactoring this to a separate function there wasn't a need to separately write "NEVER = 0".